crypto/apple.go

```diff
@@ -0,0 +1,99 @@
+package crypto
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+)
+
+const AppleRootCAPEM = `-----BEGIN CERTIFICATE-----
```

Comment:
Not objection. Feels like, at some point, we'll need to provide access to them elsewhere? But perhaps Public funcs to return a set of proper structs? E.g. `func AppleRootCACertificate() *x509.Certificate () {}` or the like (which simply convert and return that?

That said, at the moment, we don't have a need for them to exported, so perhaps not a worry for now. I can change.